### PR TITLE
Expose ignore option in CLI to ignore file patterns when calculating scope with --since option

### PR DESCRIPTION
**Pull request by pushkargupta9807**
Enables to pass files to be ignored when calculating scope with --since option .
This is helpful as we may need to pass files to be ignored for specific lage task , in addition to the general ignore patterns defined in lage.config.js. 

instance :
yarn lage lint --ignore **/some-config-files/** 
yarn lage test --ignore **/some-non-product-code-tests/**

Please note both the CLI passed ignore arguments and the arguments defined in lage.config.js files are concatenated.

### CODE REVIEW

File: packages/cli/src/commands/run/index.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ addLoggerOptions(runCommand)
   .option("--reset-cache", "resets the cache, filling it after a run")
   .option("--skip-local-cache", "skips caching locally (defaults to true in CI environments)", isRunningFromCI)
   .option("--profile [profile]", "writes a run profile into a file that can be processed by Chromium devtool")
+  .option("--ignore <ignore...>","ignores certain files when calculating the scope with `--since`", [])
```
Comment (phillipb) [inline packages/cli/src/commands/run/index.ts:36]: Need to update the help text here. Preferably something that expresses that this ignore is in addition to what's already ignored in the lage config.

**Review by phillipb — CHANGES_REQUESTED** [review]

File: packages/cli/src/commands/run/index.ts (hunk 1 above)
Comment (kenotron) [reply packages/cli/src/commands/run/index.ts:36]: I think there needs to almost be a dedicated section in the doc on these flags!! But okay, I do think we need to wordsmith this a little bit!

**Comment by kenotron** [comment]
@pushkargupta9807 - make sure to run `yarn format` if you encounter prettier issues... I really should have lint-staged run this automatically for you.

File: packages/cli/src/commands/run/index.ts (hunk 1 above)
Comment (pushkargupta9807) [reply packages/cli/src/commands/run/index.ts]: Done as suggested.

**Review by phillipb — APPROVED** [review]
LGTM

**Review by kenotron — APPROVED** [review]

**Pull request merged**